packages/client-search/src/types/SearchResponse.ts

```diff
@@ -38,4 +38,50 @@ export type SearchResponse<TObject = {}> = {
   readonly parsedQuery?: string;
   readonly userData?: any;
   readonly appliedRules?: ReadonlyArray<Readonly<Record<string, any>>>;
+
+  /**
+   * The explanation of the decompounding at query time.
+   */
+  readonly explain?: {
+    /**
+     * The explain query match.
+     */
+    readonly match: {
+      /**
+       * The explain query match alternatives.
+       */
+      readonly alternatives: ReadonlyArray<{
+        /**
+         * The alternative type.
+         */
+        readonly types: readonly string[];
+
+        /**
+         * The list of alternative words.
+         */
+        readonly words: readonly string[];
+
+        /**
+         * The number of typos.
+         */
+        readonly typos: number;
+
+        /**
+         * The offset.
+         */
+        readonly offset: number;
+
+        /**
+         * The length.
+         */
+        readonly length: number;
+      }>;
+    };
+
+    /**
+     * Query parameter reporting. Parameters are reported
+     * as a JSON object with one field per parameter.
+     */
+    readonly params?: Readonly<Record<string, any>>;
```

Comment:
I didn't managed to actually get this key back. I am missing something?

Reply:
Isn’t the type here something close to QueryParams?

Reply:
The format is something like: `{ abTests: Object, final: Array, ... }`. It may change in the near future.